python/nox.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Since this repo is no where near as complex as the Python one, you likely don't need a noxfile this complicated.

Maybe start with one of these:

https://github.com/jonparrott/nox/blob/master/nox.py
https://github.com/GoogleCloudPlatform/webapp2/blob/master/nox.py